src/views/Results.vue

```diff
@@ -39,7 +39,7 @@
 
 		<header v-if="!noSubmissions">
 			<h2>{{ t('forms', 'Responses for {title}', { title: form.title }) }}</h2>
-			<div>
+			<div id="responseActions">
```

Comment:
Please use the BEM format and no id :)
```suggestion
			<div class="response_actions">
```